packages/engine/Source/Scene/Fog.js

```diff
@@ -34,6 +34,11 @@ function Fog() {
    * @default 2.0e-4
    */
   this.density = 2.0e-4;
+  /**
+   * A scalar that impacts the visual density of fog. This value does _not_ impact the culling of terrain.
+   * Use in combination with the `density` to make fog appear more or less dense.
```

Comment:
`@type`?
`@default`?

Also, I know there are no code examples in this file, but I think it would benefit users.

I'm thinking some super simple examples fro the common properties, i.e.

```js
// Disable fog in the scene
viewer.scene.fog.enabled = false;
```

```js
// Increase fog appearance effect
viewer.scene.fog.visualDensityScalar = 1.0;
```